### ISSUE TITLE
Send a unique Peer Id to tracker

### ISSUE THREAD

**Issue by carnage** [hadouken/hadouken#214]
Currently Hadouken defaults to libtorrents default peer id/user agent this is preventing it from being white listed on some private trackers. I think that adding a line in here: https://github.com/hadouken/hadouken/blob/1928a95158b2ba726641abdc09d801e24fbd1cf9/js/core.js#L291 to set user agent to something appropriate would do the job.

**Comment by vktr** [comment]
Yes, this is by design for now. We added a unique peer id and user agent in 4.6 or 4.7 I think, but as it turns out - a totally unknown, non-whitelisted peer id is worse than the libtorrent default which is at least recognized by some private trackers :smile: 

I've been in contact with a few trackers however there's not much in the way of progress there.

**Comment by carnage** [comment]
Perhaps an option then:
- Lib torrent default
- Anon/Empty
- Hadouken
